src/main/java/com/faforever/client/game/RatingChangedLabelController.java

```diff
@@ -0,0 +1,56 @@
+package com.faforever.client.game;
+
+import com.faforever.client.fx.Controller;
+import com.faforever.client.i18n.I18n;
+import com.faforever.client.replay.Replay.PlayerStats;
+import com.faforever.client.util.RatingUtil;
+import javafx.css.PseudoClass;
+import javafx.scene.Node;
+import javafx.scene.control.Label;
+import org.springframework.beans.factory.config.ConfigurableBeanFactory;
+import org.springframework.context.annotation.Scope;
+import org.springframework.stereotype.Component;
+
+import javax.inject.Inject;
+
+@Component
+@Scope(ConfigurableBeanFactory.SCOPE_PROTOTYPE)
+public class RatingChangedLabelController implements Controller<Node> {
+  private static final PseudoClass POSITIVE = PseudoClass.getPseudoClass("positive");
+  private static final PseudoClass NEGATIVE = PseudoClass.getPseudoClass("negative");
+  public Label ratingChangedLabelRoot;
+  private final I18n i18n;
+
+  @Inject
+  public RatingChangedLabelController(I18n i18n) {
+    this.i18n = i18n;
+  }
+
+  @Override
+  public Node getRoot() {
+    return ratingChangedLabelRoot;
+  }
+
+  @Override
+  public void initialize() {
+
+  }
+
+  public void setRatingChange(PlayerStats playerStats) {
+    if (playerStats.getAfterMean() == null || playerStats.getAfterDeviation() == null) {
+      return;
+    }
+    int newRating = RatingUtil.getRating(playerStats.getAfterMean(), playerStats.getAfterDeviation());
+    int oldRating = RatingUtil.getRating(playerStats.getBeforeMean(), playerStats.getBeforeDeviation());
+
+    String changedRating = String.valueOf(newRating - oldRating);
+    if (newRating > oldRating) {
+      ratingChangedLabelRoot.setText(i18n.get("plus") + changedRating);
```

Comment:
This is assuming that whatever "plus" is, the rating should be appended to it. This may, however, not be the case in a specific locale (e.g. right-to-left languages). Also, the rating change number would not be formatted locale-sensitive.

Furthermore, Java's number formatter can take care for the leading signs (+-) for you. So you can use this after the `if` statement to cover both cases:
```
ratingChangeLabelRoot.setText(ratingChange);
```
(Note that I'd also rename the variable to `ratingChangeLabelRoot`).